pkg/controller/config/config_controller.go

```diff
@@ -195,5 +212,72 @@ func (r *ReconcileConfig) Reconcile(ctx context.Context, request reconcile.Reque
 	}
 
 	r.tracker.For(configGVK).Observe(instance)
-	return reconcile.Result{}, nil
+
+	if deleted {
```

Comment:
Should "config-controller: error establishing watches for new syncOnly:" be written to the status object? Maybe with a more user-friendly error string?